packages/swap/src/routes/swap/index.ts

```diff
@@ -66,12 +77,11 @@ router.get(
 
     let state: State;
 
-    if (swap?.egressCompletedAt) {
-      assert(swap.swapExecutedAt, 'swapExecutedAt should not be null');
```

Comment:
these asserts are still useful i think.

I only meant asserting a parent object of a child that has already been defined as redundant.

Reply:
okay i think i understand what you meant now - moved the `swapExecutedAt` check back in